model/registry/log.yaml

```diff
@@ -0,0 +1,72 @@
+groups:
+  - id: registry.log
+    type: attribute_group
+    prefix: log
+    brief: >
+      This document defines log attributes
+    attributes:
+      - id: iostream
+        stability: experimental
+        brief: >
+          The stream associated with the log. See below for a list of well-known values.
+        type:
+          allow_custom_values: true
+          members:
+            - id: stdout
+              value: 'stdout'
+              brief: 'Logs from stdout stream'
+              stability: experimental
+            - id: stderr
+              value: 'stderr'
+              brief: 'Events from stderr stream'
+              stability: experimental
+
+  - id: registry.log.file    # TODO: should we move it to the file model?
```

Comment:
Would that make sense to move this under `registry.log.*` namespace here in the registry? In order to honor the "one single and flat registry" idea.

Reply:
We don't have agreement on this [yet](https://github.com/open-telemetry/semantic-conventions/issues/887).

Reply:
We can proceed and change then later as soon as there will be any decision made. No decision is also a decision :)